### DISCUSSION TITLE
How do you know which location the Infotainment System ran scripts at?

### DISCUSSION THREAD

**Discussion by domtuazon** [xeon1989/Proton-X50-APK-Installer#17] [General]
From the Technical Discussion part, you mentioned the ff:

> Proton X50 Infotainment System are trying to run bash scripts at \b832bc61472727635baffcf25dd28e9f239273e2\d4ad0e797c738a44a9dd1b9231d201e8374dfda2.sh from USB drive.

I’m a beginner, how do you exactly find what the Infotainment System accessed on the usb drive? My guess is the Proton X70/Azkarra is doing the same thing 🤔 I was hoping to reproduce.

**Comment by xeon1989** [comment]
Hi, I'm not sure about Azkarra but Proton X70 are having different firmware, thus it's not reading the path. 
For Geely Binyue / Proton X50 GKUI19, the path is defined in update.sh in /bin.
![image](https://user-images.githubusercontent.com/17538895/191810857-0a475e40-0825-4ca7-b995-6cf674a76821.png)

**Reply by domtuazon** [reply]
I have an Azkarra and confirming that it also doesn’t work. Probably a different firmware like you said. I was hoping you’d know how they were able to find that the infotainment system was trying to execute a `*.sh` file in this exact path `\b832bc61472727635baffcf25dd28e9f239273e2`.

If we could trace which path and sh file it’s trying to access maybe this same script could work.
